README.md

```diff
@@ -134,7 +134,7 @@ We are looking for maintainers to add more parsers and to write query files for
 <!--parserinfo-->
 - [x] [bash](https://github.com/tree-sitter/tree-sitter-bash) (maintained by @TravonteD)
 - [x] [beancount](https://github.com/polarmutex/tree-sitter-beancount) (maintained by @polarmutex)
-- [ ] [bibtex](https://github.com/latex-lsp/tree-sitter-bibtex)
+- [x] [bibtex](https://github.com/latex-lsp/tree-sitter-bibtex)
```

Comment:
this is automatically edited. the cross will only be added if there is a maintainer.

Reply:
Sorry, I was going by the text before the list.

(Which I wrote, so...)